BCP00301Test.py

```diff
@@ -38,14 +41,15 @@ def perform_test_ssl(self, test, args=None):
         if args is None:
             args = []
         try:
-            ret = subprocess.run(["testssl/testssl.sh", "--jsonfile", TMPFILE, "--warnings", "off"] + args +
+            ret = subprocess.run(["testssl/testssl.sh", "--jsonfile", TMPFILE, "--warnings", "off", "--add-ca",
+                                  CERT_TRUST_CHAIN] + args +

```

Comment:
Should this be using a cert _chain_ or just be the root CA cert alone? Since the test is performing as a client shouldn't it only need the root cert, and it's the job of the server under test to include the chain from its server cert? (And the scripts could help by generating a chain for the server?)